sovtoken/sovtoken/client_authnr.py

```diff
@@ -30,9 +29,6 @@ def verify(self, sig, msg) -> bool:
 class TokenAuthNr(CoreAuthNr):
```

Comment:
Should we pass into constructor write_types, query_types, action_types as in Plenum here?

Reply:
fixed